src/vs/workbench/contrib/chat/browser/chatListRenderer.ts

```diff
@@ -208,7 +208,7 @@ export class ChatListItemRenderer extends Disposable implements ITreeRenderer<Ch
 	private getProgressiveRenderRate(element: IChatResponseViewModel): number {
 		const enum Rate {
 			Min = 5,
-			Max = 80,
+			Max = 2000,
```

Comment:
I think this won't look good because we render at the 'max' rate when responses are complete. Let me tweak it later today